src/lib/chat/helpers/getChats.js

```diff
@@ -0,0 +1,42 @@
+import * as admApi from '@/lib/adamant-api'
+
+/**
+ * Retrieve the chat messages for the current account.
+ * @param {number} startHeight
+ * @param {number} startOffset
+ * @param {boolean} recursive
+ * @returns {Promise<{messages: Message[], lastMessageHeight: number}>} Array of messages
+ */
+export function getChats(startHeight = 0, startOffset = 0, recursive = true) {
+  let allTransactions = []
```

Comment:
Why?

Reply:
@martiliones This will not work because `allTransactions` is modified by every `loadMessages()` call (see line [24](https://github.com/Adamant-im/adamant-im/blob/798ee0f529ff7db2f5b603b459836d8ef68f1b65/src/lib/chat/helpers/getChats.js#L24)).